ios/PurchasesHybridCommon/PurchasesHybridCommonTests/EntitlementInfo+HybridAdditionsTests.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  EntitlementInfo+HybridAdditionsTests.swift
+//  PurchasesHybridCommonTests
+//
+//  Created by Will Taylor on 5/24/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import Quick
+import Nimble
+@testable import PurchasesHybridCommon
+@testable import RevenueCat
+
+class EntitlementInfoHybridAdditionsTests: QuickSpec {
+
+    private static let expiresDate = "9999-01-01T00:00:00Z"
+    private static let purchaseDate = "1999-07-26T23:30:41Z"
+    private let mockEntitlementData = [
+        "expires_date": EntitlementInfoHybridAdditionsTests.expiresDate,
+        "product_identifier": "pro",
+        "purchase_date": EntitlementInfoHybridAdditionsTests.purchaseDate
+    ]
+
+    override func spec() {
+        describe("rc_dictionary") {
+            context("store") {
+                it("is properly set") {
+                    let expectations: [(Store, String)] = [
+                        (.appStore, "APP_STORE"),
+                        (.macAppStore, "MAC_APP_STORE"),
+                        (.playStore, "PLAY_STORE"),
+                        (.stripe, "STRIPE"),
+                        (.promotional, "PROMOTIONAL"),
+                        (.amazon, "AMAZON"),
+                        (.unknownStore, "UNKNOWN_STORE")
+                    ]
+
+                    for expectation in expectations {
+                        let (store, expectedDictionaryValue) = expectation
+                        let mockEntitlementInfo = self.mockEntitlementInfo(store: store)
+
+                        let dictionary = mockEntitlementInfo.dictionary
+                        expect(dictionary["store"] as? String).to(
+                            equal(expectedDictionaryValue),
+                            description: "Expected \(String(describing: dictionary["store"] as? String)) to become \(expectedDictionaryValue)."
```

Comment:
Nit: you don't need to reference `dictionary["store"]` because that will be part of the assertion message already (since that's the value you're checking. I would just do:
```suggestion
                            description: "Expected \(store) to be encoded as \(expectedDictionaryValue)."
```